### PR TITLE
Feature/for 2039 select continious scrolling

### PR DESCRIPTION
**Pull request by VladTsygankov**

### CODE REVIEW

**Comment by travist** [comment]
Very well done @VladTsygankov

File: src/components/select/editForm/Select.edit.data.js [hunk 1]
```diff
@@ -271,12 +271,30 @@ export default [
...
+    type: 'textfield',
+    input: true,
+    key: 'sort',
+    label: 'Sort Query',
+    weight: 18,
+    description: 'The sort query for results',
+    tooltip: 'User this to provide additional sorting using query parameters',
+    conditional: {
+      json: {
+        or: [
```
Comment (YuryRybakSoftensity) [inline src/components/select/editForm/Select.edit.data.js]: Please consider usage of 'in' operator here http://jsonlogic.com/operations.html#in

Comment (travist) [reply src/components/select/editForm/Select.edit.data.js]: This can be resolved in another PR. going to accept it as is.

**Comment by travist** [comment]
There is an issue with this change. If you are loading an API that does not support pagination, it will continually load the same page over and over and the list gets bigger and bigger..

**Comment by travist** [comment]
I spent some time to investigate. I also found quite a few other issues with infinite scroll. Please review these changes.

https://github.com/formio/formio.js/commit/cecabd3047013eac1ef48be8da5fd61b6f1c86a9

**Pull request merged**